201-decrypt-running-windows-vm-without-aad/azuredeploy.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the virtual machine"
+      }
+    },
+    "volumeType": {
+      "type": "string",
+      "defaultValue": "All",
+      "metadata": {
+        "description": "Type of the volume OS or Data to perform encryption operation"
+      }
+    },
+    "forceUpdateTag": {
+      "type": "string",
+      "defaultValue": "1.0",

```

Comment:
use [uniqueString(resourceGroup().id, deployment().name)]

Reply:
@bmoore-msft  As I mentioned in my comment in earlier iteration, if user does "New-AzureRmResourceGroupDeployment -Name 'ecryptVM'.." then [uniqueString(resourceGroup().id, deployment().name)] doesn't produce unique value for two different deployments. That's the reason to leave this as is.

Reply:
The current defaultValue doesn't produce anything unique either... and we're optimizing for the scenarios in which these templates are most commonly deployed 1) azure portal and 2) scripts in the repo.  Both of those methods do generate unique deployment names.